src/dispatch/report/models.py

```diff
@@ -0,0 +1,57 @@
+from datetime import datetime
+
+from typing import List, Optional
+
+from sqlalchemy import Column, DateTime, Integer, String, ForeignKey
+from sqlalchemy.orm import relationship
+
+from dispatch.database import Base
+from dispatch.models import DispatchBase
+
+from .enums import ReportTypes
+
+
+class Report(Base):
+    id = Column(Integer, primary_key=True)
+    created_at = Column(DateTime, default=datetime.utcnow)
+    conditions = Column(String)
```

Comment:
This feels likes like we just combined two models (instead of making something more generic/extensible).

Reply:
Yeah, perhaps we could store the report details in a JSON field?